study.md

```diff
@@ -46,4 +55,6 @@ What are some advantages and disadvantages of async?
 
 ```md
 <!-- your answer here -->
+Pro: prevents overall function of a webpage from slowing down
+con: can add complexity to code and make it harder to debug? Not entirely sure on this
 ```
```

Comment:
You are totally write. the con you're referring to is also known as callback hell. If we want things to happen synchronously in JS, we usually do this with callback functions. Then sometimes those have callback function etc.. This can be a pain to debug sometimes! Good answers